tests/test_local_invites.py

```diff
@@ -35,7 +42,7 @@ def test_invite_to_dm(self) -> None:
         room_id = self.helper.create_room_as(
             self.user_a, is_public=False, tok=self.access_token
         )
-        assert room_id, "Room created"
+        assert room_id, "Room not created"
```

Comment:
Wouldn't now the error be: "Assertion Room not created failed"? That seems backwards to me.